### ISSUE TITLE
Uncaught Error: Adding a pane item with URI '/vobs/EFJ_Products/main/trunk/host/apps/ccf/src/Comp...

### ISSUE THREAD

**Issue by github-sjacobs** [susisu/tree-view-panes#7]
[Enter steps to reproduce:]
Open Files list gets out of sync. In this case it was listing a file that was not open and I selected it.

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: "CentOS Linux
**Thrown From**: [tree-view-panes](https://github.com/susisu/tree-view-panes) package 0.3.5

### Stack Trace

Uncaught Error: Adding a pane item with URI '/vobs/EFJ_Products/main/trunk/host/apps/ccf/src/ComponentMgr.cpp' that has already been destroyed

```
At /usr/share/atom/resources/app/src/pane.js:493

Error: Adding a pane item with URI '/vobs/EFJ_Products/main/trunk/host/apps/ccf/src/ComponentMgr.cpp' that has already been destroyed
    at Pane.module.exports.Pane.addItem (/usr/share/atom/resources/app/src/pane.js:493:21)
    at Pane.module.exports.Pane.activateItem (/usr/share/atom/resources/app/src/pane.js:468:20)
    at PaneController.activateItem (/packages/tree-view-panes/lib/pane-controller.js:186:15)
    at HTMLLIElement.ItemController.onClick (/packages/tree-view-panes/lib/item-controller.js:163:19)
```

### Commands

```
     -8:05.8.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -7:56.3.0 core:confirm (input.hidden-input)
     -2:21 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -2:16.8.0 core:confirm (input.hidden-input)
     -0:54.8.0 core:copy (input.hidden-input)
     -0:50.8.0 core:paste (input.hidden-input)
     -0:45.6.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.1 
code-dark-ui 0.7.0 
counter 0.3.5 
file-icons 2.1.7 
geiger 2.1.0 
highlight-line 0.12.0 
highlight-selected 0.13.1 
language-lua 0.9.11 
line-count-status 0.0.3 
line-ending-converter 1.3.2 
markdown-preview-plus 2.4.9 
minimap 4.28.2 
minimap-bookmarks 0.4.2 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.2 
minimap-highlight-selected 4.6.1 
sort-lines 0.14.0 
space-tab 0.4.2 
symbols-tree-view 0.14.0 
symbols-tree-view-fix 0.13.3 
sync-settings 0.8.2 
tree-view-autoresize 2.1.0 
tree-view-panes 0.3.5 
tree-view-sort 0.7.0 
tron-legacy-syntax 0.5.0 
tron-legacy-ui 0.6.0 
Visual-Studio-2015-Dark-Theme 2.4.2 
```

**Comment by susisu** [comment]
Thank you for reporting so many times!
I've never got such kind of errors and I guess some package you are using is conflicting with the package, but I don't know actually. 🤔 

Anyway, this kind of out-of-sync problems may be fixed together with #5 so please wait...

**Comment by susisu** [comment]
I decided I won't fix this, because adding guards to check the item is already destroyed or not for all the operations costs too much, and it should not happen if the state is sound.
